new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -475,6 +475,27 @@ public static String getPodCreationTimestamp(String namespace, String labelSelec
     }
   }
 
+  /**
+   * Get the weblogic.domainRestartVersion label from a given pod.
+   *
+   * @param namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
+   * @param podName  name of the pod
+   * @return value of weblogic.domainRestartVersion label, null if unset or the pod is not available
+   * @throws ApiException when there is error in querying the cluster
+   */
+  public static String getPodRestartVersion(String namespace, String labelSelector, String podName)
```

Comment:
Can we make this as an Action through TestAction.java, so that all MII test can use it, to verify and upgrade the Domain Restart Version

Reply:
sure.